package.json

```diff
@@ -3,47 +3,37 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "test": "vue-cli-service test:unit --verbose -u"
+    "web:serve": "cd webapp && npm run serve",
+    "web:build": "cd webapp && npm run build",
+    "web:lint": "cd webapp && npm run lint",
+    "web:test": "cd webapp && npm test",
+    "backend:start": "cd backend && npm run build && npm start",
+    "backend:lint": "cd backend && npm run lint",
+    "backend:test": "cd backend && npm test"
   },
   "dependencies": {
     "@babel/core": "^7.7.2",
-    "acorn": "^7.1.0",
-    "acorn-jsx": "^5.1.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "core-js": "^3.3.2",
-    "vue": "^2.6.10",
-    "vue-router": "^3.1.3"
+    "babel-core": "^7.0.0-bridge.0"
```

Comment:
Isn't `babel-core` also a `devDependency`? :thinking: or do you need it for something at runtime?

Reply:
I have absolutely no idea